tests/framework/builder.py

```diff
@@ -218,11 +220,9 @@ def build_from_snapshot(self,
                                          tapname=iface.tap_name)
         if utils.compare_versions(vm.firecracker_version, '1.0.0') > 0:
             if uffd_path:
-                jailed_uffd = vm.copy_to_jail_ramfs(uffd_path) if use_ramdisk \
-                    else vm.create_jailed_resource(uffd_path)
```

Comment:
why did we remove this? don't we need the uds path to be inside the jail?

Reply:
Yes, we do need to have the UDS inside the jail, but it must be already present in the jail before issuing the command to build the microvm from snapshot, because the page fault handler needs to bind it and accept incoming connections.